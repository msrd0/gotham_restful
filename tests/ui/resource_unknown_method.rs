#[macro_use] extern crate gotham_restful;

#[derive(Resource)]
#[resource(read_any)]
struct FooResource;

#[read_all(FooResource)]
fn read_all()
{
}

fn main()
{
}
