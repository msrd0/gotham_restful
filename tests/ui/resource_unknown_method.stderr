error: Unknown method: `read_any'
 --> $DIR/resource_unknown_method.rs:4:12
  |
4 | #[resource(read_any)]
  |            ^^^^^^^^

error[E0277]: the trait bound `FooResource: Resource` is not satisfied
 --> $DIR/resource_unknown_method.rs:7:1
  |
7 | #[read_all(FooResource)]
  | ^^^^^^^^^^^^^^^^^^^^^^^^ the trait `Resource` is not implemented for `FooResource`
  |
  = help: see issue #48214
  = note: this error originates in an attribute macro (in Nightly builds, run with -Z macro-backtrace for more info)
